src/hooks/core/websocket/types.d.ts

```diff
@@ -6,20 +6,31 @@ export type WebsocketUserDataType = {
   image?: string;
 };
 
+export type BetterSocialMessageType =
+  | 'follow_user'
+  | 'follow_topic'
+  | 'new_topic_post'
+  | 'add_member_to_group'
+  | 'remove_member_from_group'
+  | 'change_channel_detail'
+  | 'leave_group';
+
+export type MessageType =
+  | 'regular'
+  | 'system'
+  | 'deleted'
+  | 'reply'
+  | 'reply-prompt'
+  | 'notification-deleted';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the definition of `MessageType` also enhances code readability and maintainability. It's always better to have such types defined at one place which can be used across the application.

```typescript
export type MessageType =
  | 'regular'
  | 'system'
  | 'deleted'
  | 'reply'
  | 'reply-prompt'
  | 'notification-deleted';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->